### DISCUSSION TITLE
is there a requirements management feature in allure?

### DISCUSSION THREAD

**Discussion by tadeek** [allure-framework/allure2#2520] [Questions & Support] [answered]
I would like to have a list of requirements and map them to tests, so i can see how many requirements are met by current system state, and which are not.

**Comment by zhivanshe** [comment, marked as answer]
Hi, @tadeek!
We plan to implement the connection of tests with requirements. 
Something will already appear by the end of Q'2 24. In Q'3 24, we plan to release an MLP (minimum viable product) version of the product.
